frontend/src/components/collective/UnpaidExpenseItem.js

```diff
@@ -0,0 +1,46 @@
+import React, {PropTypes} from 'react';
+
+import Currency from '../Currency';
+
+const UnpaidExpenseItem = ({ 
```

Comment:
Same here, I think it would be nicer to have just a `ExpenseItem.js` component and within which it should handle the different states. Or if there is really a lot of difference, we should have a super component "ExpenseItem.js" and sub components for each state.